security/sec-iride/pom.xml

```diff
@@ -73,7 +73,9 @@
 
         <!-- Dependencies -->
         <gs.version>2.8-SNAPSHOT</gs.version>
+        <freeemarker.version>2.3.18</freeemarker.version>
         <spring.version>3.1.4.RELEASE</spring.version>
+        <spring-security.version>3.1.4.RELEASE</spring-security.version>

```

Comment:
Ok, I had the version of Spring Security match that of Spring Framework because of some errors occuring during test execution, after having introduced Spring Test; but those errors seem to have been resolved.